app/controllers/api/public/v1/providers/locations_controller.rb

```diff
@@ -0,0 +1,27 @@
+module API
+  module Public
+    module V1
+      module Providers
+        class LocationsController < API::Public::V1::ApplicationController
+          def index
+            render jsonapi: locations, class: API::Public::V1::SerializerService.execute
+          end
+
+        private
+
+          def locations
+            @locations ||= provider.courses.find_by(course_code: params[:course_id])&.sites
```

Comment:
```suggestion
            @locations ||= provider.courses.find_by(course_code: params[:course_code])&.sites
```

I think we would want to use course `code` over `id`

https://github.com/DFE-Digital/teacher-training-api/blob/4de4daa71609dcd6cadf0061fea1af4702fcae3d/spec/docs/locations_spec.rb#L4

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1513/commits/3aa94e6bf170121fa9209ded629dca486c401c4e